.github/workflows/check-outdated-content.yml

```diff
@@ -0,0 +1,159 @@
+# Contribution by Cloud Native Guatemala folks :)
+# This workflow will check if a localized content is outdated or not
+# by comparing English content in the old branch and the latest branch.
+---
+name: Check outdated content
+
+on:
+  pull_request:
+
+
+jobs:
+  check-outdated-content:
+    name: Check outdated content
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout 
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0 # fetch all history for all tags and branches
+    - name: Detecting pending translations
+      shell: bash
+      run: |
+        ##### DEBUG section, this will be removed later ###########
+        ls -al
+        git status
+        git branch
+        
+        # Default environment variables 
+        echo "GITHUB_REF: $GITHUB_REF"
+        echo "Extract branch: ${GITHUB_REF#refs/}"
+        
+        # `github` context information
+        echo "(DEBUG) github.ref: ${{github.ref}}"
+        echo "(DEBUG) github.head_ref: ${{github.head_ref}}"
+        echo "(DEBUG) github.base_ref: ${{github.base_ref}}"
+        #####################################################
+
+        OUTPUT_DIR="./outdated"
+        CONTENT_DIR="website/content"
+        languages=("es" "zh")
```

Comment:
I am wondering how we make sure that we have minimal maintenance, adding new translations in the future. We could for example look at the repository and check the folder names https://github.com/cncf/tag-env-sustainability/tree/main/website/content - we have a folder `en` & `zh` & `es` &`de`, we establish a base with `en` (we dont need to open a PR for en since its already merged). This could eliminate referencing languages in the script. 

Assigning people over groups could work this way too. We assign `tag-env-translators-<LANGUAGE REFERENCE>` so for example `tag-env-translators-es`.

Reply:
Yes, we could work on it.

Reply:
+1, I've added it as one of the potential improvement in https://github.com/cncf/tag-env-sustainability/pull/407#issuecomment-2150667464
@sergioarmgpl @Dianmz if you agree with this suggestion, can you please create a task to track this as an improvement?

Reply:
We create this issue to work on this improvement https://github.com/cncf/tag-env-sustainability/issues/429